### DISCUSSION TITLE
Add VT-ADL: a new Anomaly Detector

### DISCUSSION THREAD

**Discussion by innat** [open-edge-platform/anomalib#188] [Feature Requests]
**Is your feature request related to a problem? Please describe.**
  - [VT-ADL: A Vision Transformer Network for Image Anomaly Detection and Localization](https://arxiv.org/pdf/2104.10036.pdf)

![image](https://user-images.githubusercontent.com/17668390/160456734-727073b3-00f0-44db-9cf1-eaee8c2138ed.png)

**Describe the solution you'd like**
  - [Official Code](https://github.com/pankajmishra000/VT-ADL). 

**Additional context**
  - Did a sneak peek over the paper. Unfortunately didn't find any comparison with models like patch-core or fat flow etc.

**Comment by samet-akcay** [comment]
We are currently adding UniNet and Dinomaly papers from CVPR 25, GLASS from ECCV 24, which are the sota approaches. This paper is therefore out of scope.
